packages/server/server.ts

```diff
@@ -58,6 +61,15 @@ export async function startServer(isDev: boolean, port: number) {
 		}
 	});
 
+	app.use('/api/themes', async (req, res) => {
+		try {
+			await themeApiHandler(req, res);
+		} catch (e) {
+			console.log(e);
```

Comment:
Имеет смысл заменить на:
```suggestion
			console.error(e);
```

Если есть желание оставить логирование ошибок